docs/providers/aws/guide/functions.md

```diff
@@ -216,6 +216,8 @@ provider:
       baseimage:
         path: ./path/to/context
         file: Dockerfile.dev
+        buildArgs:
```

Comment:
It would be great to add an extra explanation to that in the above section of the docs in a similar way as other properties are described.